src/olympia/lib/settings_base.py

```diff
@@ -1942,3 +1942,5 @@ def get_sentry_release():
 ADZERK_URL = f'https://e-{ADZERK_NETWORK_ID}.adzerk.net/api/v2'
 ADZERK_IMPRESSION_URL = f'https://e-{ADZERK_NETWORK_ID}.adzerk.net/i.gif?'
 ADZERK_IMPRESSION_TIMEOUT = 60  # seconds
+ADZERK_CLICK_URL = f'https://e-{ADZERK_NETWORK_ID}.adzerk.net/r?'
+ADZERK_CLICK_TIMEOUT = 60 * 60  # seconds
```

Comment:
Why is this 60 * 60, whereas `ADZERK_IMPRESSION_TIMEOUT` is just 60?

Reply:
One hour sounds like more than enough. I'm good with it.